packages/ibm-products/src/components/DecoratorDualButton/DecoratorDualButton.mdx

```diff
@@ -12,6 +12,11 @@ import * as DecoratorDualButtonStories from './DecoratorDualButton.stories';
 - [Example setLabelTitle()](#example-setlabeltitle)
 - [Component API](#component-api)
 
+## Deprecated
```

Comment:
can you update these headers to `Deprecation notice` in order to align with the new `deprecationNotice` prop in `StoryDocsPage`?